next.config.js

```diff
@@ -3,32 +3,35 @@ const {
   PHASE_PRODUCTION_BUILD,
 } = require("next/constants");
 
-const can = require("./canopy.config");
+const config = require("./canopy.config");
+const canopy = require("./services/canopy");
 
 module.exports = (phase) => {
+  /**
+   *
+   */
   const isDev = phase === PHASE_DEVELOPMENT_SERVER;
   const isProd = phase === PHASE_PRODUCTION_BUILD;
 
-  const { config, dev } = can;
+  const { prod, dev } = config;
+
+  config.environment = (() => {
+    if (isDev) return dev;
+    if (isProd) return prod;
+  })();
 
   const env = {
-    collection: (() => {
-      if (isDev) return dev.collection;
-      if (isProd) return config.collection;
-    })(),
-    hero: (() => {
-      if (isDev) return dev.hero;
-      if (isProd) return config.hero;
-    })(),
-    metadata: (() => {
-      if (isDev) return dev.metadata;
-      if (isProd) return config.metadata;
-    })(),
-    title: (() => {
-      if (isDev) return dev.title;
-      if (isProd) return config.title;
-    })(),
+    ...config.globals,
+    ...config.environment,
   };
 
+  /**
+   *
+   */
+  canopy.buildCanopy(env);
+
+  /**
+   *
+   */
```

Comment:
Again, is there a reason why we have these empty comments? 

```suggestion
  canopy.buildCanopy(env);
```